### PR TITLE
[Fix](mlu-ops):fix racecheck error.

### PR DESCRIPTION
**Pull request by mahxn0**
Thanks for your contribution and we appreciate it a lot. :rocket::rocket:

## 1. Motivation

fix racecheck error.

## 2. Modification

kernels/generate_proposals_v2/generate_proposals_v2_nms_utils.h
kernels/generate_proposals_v2/generate_proposals_v2_union1_500.mlu
kernels/rotated_feature_align/rotated_feature_align_block.mlu
kernels/voxelization/voxelization_kernel.mlu

### CODE REVIEW

File: kernels/voxelization/voxelization_kernel.mlu [hunk 1]
```diff
@@ -197,13 +197,14 @@ __mlu_global__ void mluDynamicVoxelize(const float *points,
...
-      FLOOR_ALIGN(MAX_NRAM_SIZE / split_num / sizeof(float), NFU_ALIGN_SIZE);
+  // voxel_size + coors_range need 36 bytes on nram.
+  const int32_t deal_num = FLOOR_ALIGN(
+      (MAX_NRAM_SIZE - 36) / split_num / sizeof(float), NFU_ALIGN_SIZE);
   const int32_t repeat = points_per_core / deal_num;
   const int32_t rem = points_per_core % deal_num;
   const int32_t ping_pong_gap = 3 * deal_num * sizeof(float);
 
-  int8_t *points_x = nram_buffer;
+  int8_t *points_x = nram_buffer + 9 * sizeof(float);
```
Comment (PetrelYy) [inline kernels/voxelization/voxelization_kernel.mlu:207]: 为啥是9和float 长度?

Comment (mahxn0) [reply kernels/voxelization/voxelization_kernel.mlu:207]: 前面的修改处有备注，是因为两个短向量占用了36个bytes，但是nram没有正确的偏移

**Review by PetrelYy — APPROVED** [review]
LGTM

File: kernels/voxelization/voxelization_kernel.mlu (hunk 1 above)
Comment (mahxn0) [reply kernels/voxelization/voxelization_kernel.mlu]: fixed

File: kernels/voxelization/voxelization_kernel.mlu [hunk 2]
```diff
@@ -197,13 +197,14 @@ __mlu_global__ void mluDynamicVoxelize(const float *points,
...
-      FLOOR_ALIGN(MAX_NRAM_SIZE / split_num / sizeof(float), NFU_ALIGN_SIZE);
+  // voxel_size + coors_range need 36 bytes on nram.
+  const int32_t deal_num = FLOOR_ALIGN(
+      (MAX_NRAM_SIZE - 36) / split_num / sizeof(float), NFU_ALIGN_SIZE);
   const int32_t repeat = points_per_core / deal_num;
   const int32_t rem = points_per_core % deal_num;
   const int32_t ping_pong_gap = 3 * deal_num * sizeof(float);
 
-  int8_t *points_x = nram_buffer;
+  int8_t *points_x = nram_buffer + 36;
```
Comment (Unireverse) [inline kernels/voxelization/voxelization_kernel.mlu:207]: 这个36哪里来的

**Review by DanieeelLiu — APPROVED** [review]

**Review by DanieeelLiu — COMMENTED** [review]
LGTM

**Comment by Unireverse** [comment]
LGTM

**Review by Unireverse — APPROVED** [review]

**Pull request merged**